src/main/java/org/ow2/proactive/workflow_catalog/rest/controller/WorkflowController.java

```diff
@@ -117,4 +118,11 @@ public PagedResources list(@PathVariable Long bucketId,
         return workflowService.listWorkflows(bucketId, query, pageable, assembler);
     }
 
+    @ApiOperation(value = "Delete a workflow", notes = "Returns the deleted Workflow's metadata")
```

Comment:
Would be nice to explain that all associated workflow revisions are removed too even if it seems implicit.